common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -0,0 +1,18 @@
+.. _common-geofencing-landing-page:
+
+==========
+GeoFencing
+==========
+
+ArduPilot supports several types of GeoFencing failsafes to prevent the vehicle from traveling higher or further than desired or into unwanted areas. Types of GeoFences supported varies by vehicle.
```

Comment:
Is it really several fence failsafes or is it really just a single failsafe that works with the various types of fences AP supports?  Anyway, I'm being a bit pedantic and I don't feel too strongly about it.

Reply:
I can change...no problem...."supports failsafes generated by several types of Fences"